handler/handler.go

```diff
@@ -93,6 +94,48 @@ func (h *Handler) ReadAnnotations(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (h *Handler) WriteAnnotations(w http.ResponseWriter, r *http.Request) {
+	w.Header().Add("Content-Type", "application/json")
+
+	contentUUID := vestigo.Param(r, "uuid")
+	tID := tidutils.GetTransactionIDFromRequest(r)
+	ctx := tidutils.TransactionAwareContext(context.Background(), tID)
+
+	writeLog := log.WithField(tidutils.TransactionIDKey, tID).WithField("uuid", contentUUID)
+
+	if err := validateUUID(contentUUID); err != nil {
+		writeLog.WithError(err).Error("Invalid content UUID")
+		writeMessage(w, fmt.Sprintf("Invalid content UUID: %v", contentUUID), http.StatusBadRequest)
+		return
+	}
+
+	var draftAnnotations []annotations.Annotation
+	err := json.NewDecoder(r.Body).Decode(&draftAnnotations)
+	if err != nil {
+		writeLog.WithError(err).Error("Unable to unmarshal annotations body")
+		writeMessage(w, fmt.Sprintf("Unable to unmarshal annotations body: %v", err.Error()), http.StatusBadRequest)
+		return
+	}
+
+	writeLog.Info("Canonicalizing annotations...")
+	draftAnnotations = h.c14n.Canonicalize(draftAnnotations)
+
+	writeLog.Info("Writing to annotations RW...")
+	err = h.annotationsRW.Write(ctx, contentUUID, draftAnnotations)
+	if err != nil {
+		writeLog.WithError(err).Error("Error in writing draft annotations")
+		writeMessage(w, fmt.Sprintf("Error in writing draft annotations: %v", err.Error()), http.StatusInternalServerError)
+		return
+	}
+
+	w.WriteHeader(http.StatusOK)
```

Comment:
We need to write the canonicalized annotations as the response body.

Reply:
You are right. I forgot about it

Reply:
Fixed